src/Raven.Server/Documents/Queries/AbstractQueryRunner.cs

```diff
@@ -182,9 +182,9 @@ public BulkOperationCommand(T command, bool retieveDetails, Func<T, IBulkOperati
                 _getDetails = getDetails;
             }
 
-            public override int Execute(DocumentsOperationContext context)
+            protected override int ExecuteCmd(DocumentsOperationContext context)
             {
-                var count = _command.Execute(context);
+                var count = _command.Execute(context, null);
```

Comment:
This should be using `Execute` directly and passing the `recorder` to `_command`

Reply:
Is there any reason not to do as suggested?

Reply:
done